suricata.yaml.in

```diff
@@ -1777,6 +1777,7 @@ spm-algo: auto
 # Suricata is multi-threaded. Here the threading can be influenced.
 threading:
   set-cpu-affinity: no
+  autopin: no
```

Comment:
Is `autopin` or `pinning` common language for this? I just ask, cause its new to Suricata, and the option is called `cpu-affinity`, not `cpu-pinning`.

Reply:
Good point though cpu pinning is often used. It's relatively interchangeable term though somewhere I found that it actually means strict configuration of the thread to the given CPU core whereas affinity is rather the highly preferred variant. And CPU pinning is what we actually do in Suri.